src/app/view-config-utils.js

```diff
@@ -0,0 +1,334 @@
+/* eslint-disable no-plusplus */
+import {
+  COORDINATION_TYPES, DEFAULT_COORDINATION_VALUES,
+  COMPONENT_COORDINATION_TYPES,
+} from './state/coordination';
+
+/**
+ * Generate a new scope name which does not
+ * conflict / overlap with a previous scope name.
+ * @param {string[]} prevScopes Previous scope names.
+ * @returns {string} The new scope name.
+ */
+export function getNextScope(prevScopes) {
```

Comment:
Right they just need to be unique within the coordination object. So really they could be `String(Math.random())` or `uuidv4()` or something. I just thought it may be good to make them more human-readable and memorable since eventually we will want to expose a UI to update the coordination.